src/App.jsx

```diff
@@ -1,13 +1,16 @@
 import './App.css'
 import TodoList from './TodoList'
 import AddTodoForm from './AddTodoForm'
+import { useState } from 'react'
 
 function App() {
+  const [ newTodo, setNewTodo ] = useState('');
 
   return (
     <>
       <h1>Todo List</h1>
-      <AddTodoForm />
+      <AddTodoForm onAddTodo={setNewTodo} />
+      <p>{`${newTodo}`}</p>
```

Comment:
nit: Since `newTodo` is already a string, you can just do `{newTodo}`, don't need to use a string literal

Reply:
okay noted! thanks for the feedback! 😸